src/Twig.php

```diff
@@ -104,7 +104,9 @@ public function fetch($template, $data = [])
      */
     public function render(ResponseInterface $response, $template, $data = [])
     {
-         return $response->getBody()->write($this->fetch($template, $data));
+         $response->getBody()->write($this->fetch($template, $data));
```

Comment:
Hello

So, 
```php
return $response->getBody()->write(...)
```
is not a solution as it doesn't return an ResponseInterface.
And IIUC, you suggest 
```php
return $response->write(....)
```
instead of
```php
$response->getBody()->write(...)
return $response;
```

I agree that the result is correct in both cases, but I'm not sure to see the differences (and find mine more explicit as it is the response's body that you're modifying) :(